R/derive_param_bor.R

```diff
@@ -475,29 +475,9 @@ derive_param_bor <- function(dataset,
   # aval_fun(AVALC): Execute aval_fun ----
```

Comment:
Should the assertions be dropped in the derive functions that check aval_fun(), i.e. assert_function(arg = aval_fun)?

Reply:
Yes, good point. I'll remove them.